test/bench/bench.cpp

```diff
@@ -40,6 +40,12 @@ void benchmark_parse(
 {
     const auto engine = create_fn();
 
+    // Pre-run for validation
```

Comment:
Not sure if this is preferred or just a check in the for loop.

Reply:
I think it is fine for now. Just you can omit {}.

Reply:
Copied the style from `if (!func_idx)` but I guess because had that 2 lines of message.

Reply:
Changed.